USAGE.md

```diff
@@ -2,7 +2,7 @@
 
 ## Requirements
 * [Java 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) is required for compiling the SDK. 
-  Java 1.8 and Java 11 application usage is now supported.
+* Supports applications built on Java 11. Verified by [java-manta-test-harness](https://github.com/joyent/java-manta-test-harness).
```

Comment:
Minor nit: in different places, we're using "built with Java 11" and "built on Java 11". Let's be consistent and stick with "built with Java 11".

Reply:
Noted.